lib/node_modules/@stdlib/blas/base/gscal/test/test.main.js

```diff
@@ -139,17 +188,99 @@ tape( 'the function supports specifying a stride', function test( t ) {
 	t.end();
 });
 
-tape( 'if provided a `stride` less than or equal to `0`, the function returns `x` unchanged', function test( t ) {
+tape( 'the function supports specifying a stride (accessors)', function test( t ) {
+	var expected;
+	var x;
+
+	x = toAccessorArray([
+		2.0,  // 0
+		-3.0,
+		-5.0, // 1
+		7.0,
+		6.0   // 2
+	]);
+	expected = toAccessorArray([
+		10.0,  // 0
+		-3.0,
+		-25.0, // 1
+		7.0,
+		30.0   // 2
+	]);
+
+	gscal( 3, 5.0, x, 2 );
+	t.deepEqual( x, expected, 'returns expected value' );
```

Comment:
I believe we've discussed this before, but this does not behavior as you think it does. `t.deepEqual` is not accessor aware.